explorer/app/config/model.ts

```diff
@@ -107,8 +107,32 @@ export interface ColumnConfig<
   };
   componentConfig: ComponentConfig<C, T>;
   hiddenColumn?: boolean;
+  width: GridTrackSize;
 }
 
+export type GridTrackAuto = "auto"; // Dimension specifying the track's maximum of the largest max-content size of the items in that track.
+export type GridTrackFlex = `${number}fr`; // Dimension specifying the track's flex factor; unit in "fr".
+export type GridTrackLength = `${number}px`; // Dimension specifying the track's (fixed) width; unit in "px".
+
+/**
+ * A min and max dimension specifying a size range greater than or equal to min and less than or equal to max.
+ * As a maximum, a GridTrackFlex value sets the track's flex factor and is invalid as a minimum.
+ */
+export interface GridTrackMinMax {
+  max: GridTrackAuto | GridTrackFlex | GridTrackLength;
+  min: GridTrackAuto | GridTrackLength;
+}
+
+/**
+ * A selection of possible types of track sizing values of each track (column).
+ * See https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template-columns#values.
+ */
+export type GridTrackSize =
```

Comment:
GridTrackSize is not an exhaustive list of options, but a good first round of possible ways to define the track size.